modules/axtask/src/api.rs

```diff
@@ -142,6 +142,20 @@ pub fn set_priority(prio: isize) -> bool {
     current_run_queue::<NoPreemptIrqSave>().set_current_priority(prio)
 }
 
+/// Set the affinity for the current task.
+/// [`cpumask::CpuMask`] is used to specify the CPU affinity.
+/// Returns `true` if the affinity is set successfully.
+///
+/// TODO: support set the affinity for other tasks.
+pub fn set_current_affinity(cpumask: CpuMask) -> bool {
```

Comment:
Update arceos_api?

Reply:
ok, should we pass a `u128` or just pass `cpumask::CpuMask` as a parameter